src/main/java/edu/ie3/datamodel/io/factory/EntityData.java

```diff
@@ -36,8 +36,8 @@
   private static final Logger logger = LoggerFactory.getLogger(EntityData.class);
   private static final GeoJsonReader geoJsonReader = new GeoJsonReader();
 
-  private final Map<String, String> fieldsToAttributes;
-  private final Class<? extends UniqueEntity> entityClass;
+  protected final Map<String, String> fieldsToAttributes;
```

Comment:
![Nope](https://media1.tenor.com/images/583ac7bd474a80ee73b3f891049f7aa7/tenor.gif?itemid=12298561)
```suggestion
  pivate final Map<String, String> fieldsToAttributes;
```

use `getFieldsToValues()` instead